packages/fxa-payments-server/src/components/Breadcrumbs/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import './index.scss';
+
+type BreadcrumbsProps = {
+  children: any,
+  className?: string,
+};
+
+export const Breadcrumbs = ({
+  children,
+  className = 'breadcrumbs',
+}: BreadcrumbsProps) => {
+  return (
+      <ol className={className}>{children}</ol>
```

Comment:
If this is all that the breadcrumbs component does, you could just leave it out and use `<ol>` directly in AppLayout. Alternatively, you could move the links and the code to access `AppContext` into here, too